bot_commands.py

```diff
@@ -1,66 +1,65 @@
 from datetime import datetime
+import utils
 
 from random import randint
-import utils
 
 SCHEDULE = utils.load_schedule()
 
 GENERIC_REPLIES = [
     'oh really!?',
-    'You look great today',
     'Sounds interesting, tell me more',
-    'That is a good thought',
-    'Much wow, many excite',
-    'Have you ever tried that re-usable bamboo paper towel?',
+    'That is definitely a good thought',
     'How about that local sport team?',
-    'Awesome right?',
     'I completely agree',
-    'Sprechen Sie deutsch?',
+    'Sprechen Sie Deutsch?',
     'Great weather today!',
-    'Did you get stuck in traffic this morning?',
-    'I like your haircut!',
-    'That sounds pleasant',
     'I am a robot',
     'I completely agree',
-    'Nice venue',
-    'Are you participating in the Olympia?',
-    'I would like to go to the beach.',
+    'How about this WiFi?',
+    'Are you participating in the Olympia? Its lit',
+    'Are there beaches in Berlin?.',
     'I really enjoyed the previous talk on the Mainchain stage.',
-    'Hungry for apples?',
+    'Hungry for apples or bananas?',
     'I only have finitely many distinct replies.',
-    'All signs point to yes',
-    'Lemonade is the German word for pop which is the Canadian word for soda',
     'Most likely',
 ]
 
+
 def start(bot, update):
     bot.send_message(
         chat_id=update.message.chat_id,
         text="I'm the dAppCon Schedule bot! Type /help for a list of commands. "
              "\nHave a question for the ongoing talks @dappcon_berlin? "
-             "Please head to https://www.sli.do/ and enter the code 9189 to ask your question!"
+             "\nPlease head to https://www.sli.do/ and enter one of the following codes:"
+             "\n - The legally - compliant DAO: More hype than substance Code: #6737"
+             "\n - Look at my flashy colors and rounded corners! Code: #B398"
+             "\n - Epicenter Live Code: #K367"
+             "\n - Tales of governance: DAOs, Swarms and Anarchic systems Code: #Q749"
+             "\n - AMA - Joseph Lubin Code: #Q293"
+             "\n - How to measure success? Code: #D624"
     )
 
+
 def echo(bot, update):
     random_index = randint(0, len(GENERIC_REPLIES))
     reply = GENERIC_REPLIES[random_index]
-    # reply = "Have a question for the ongoing talks @dappcon_berlin?\n" \
-    #         "Please head to https://www.sli.do/ and enter the code 9189 to ask your question!"
     bot.send_message(
         chat_id=update.message.chat_id,
         text=reply
     )
 
+
 def now(bot, update):
     bot.send_message(
         chat_id=update.message.chat_id,
         text=str(datetime.now())
     )
 
+
 def next(bot, update):
     now = datetime.now()
 
-    build = [b for b in SCHEDULE if b.location == 'Buidl Room' if b.start >= now]
+    build = [b for b in SCHEDULE if b.location in ['Rinkeby Room', "Kovan Room", "Ropsten Room"] if b.start >= now]
```

Comment:
It would be nice to use a set here!
```suggestion
    build = [b for b in SCHEDULE if b.location in {'Rinkeby Room', "Kovan Room", "Ropsten Room"} if b.start >= now]
```